Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -370,7 +370,9 @@ WarpX::computePhi (const amrex::Vector<std::unique_ptr<amrex::MultiFab> >& rho,
         // With embedded boundary: extract EB info
         MLEBNodeFDLaplacian linop( {Geom(lev)}, {boxArray(lev)}, {DistributionMap(lev)}, info, {&WarpX::fieldEBFactory(lev)});
 
-#ifndef WARPX_DIM_RZ
+#ifdef WARPX_DIM_RZ
+            linop.setSigma(0._rt, 1._rt);
```

Comment:
Fixer syntax to address compilation error:
```suggestion
            linop.setSigma({0._rt, 1._rt});
```
- [ ] testing locally